scripts/v8-patches/v8-9.4.146.17.patch

```diff
@@ -175,3 +175,15 @@
  #if V8_LIBC_MSVCRT
  #include <intrin.h>  // __cpuid()
  #endif
+--- BUILD.original.gn	2021-12-01 14:44:33.003896815 +0000
++++ BUILD.gn	2021-12-01 14:44:47.215857300 +0000
+@@ -53,8 +53,7 @@
+   }
+ }
+ 
+-use_x86_x64_optimizations =
+-    (current_cpu == "x86" || current_cpu == "x64") && !is_ios
++use_x86_x64_optimizations = false
```

Comment:
Disables use of intrinsics in zlib. There's likely a better way to do this, but I don't understand gn and how to make it pass options through to here.